app/services/show_attendee_phone_number_for_user.rb

```diff
@@ -0,0 +1,16 @@
+class ShowAttendeePhoneNumberForUser
+    def initialize(current_user, attendance)
+        @current_user = current_user
+        @attendance = attendance
+    end
+
+    def call
+        # logic operating on @current_user and @attendance
+        # returns true/false
+        @current_user.host? &&
+            @attendance.todo? &&
```

Comment:
One condition per line, good for line-by-line git updates in the future? Or unnecessary?